go/libraries/doltcore/sqle/enginetest/dolt_server_test.go

```diff
@@ -336,3 +347,92 @@ func newConnection(t *testing.T, serverConfig sqlserver.ServerConfig) (*dbr.Conn
 	sess := conn.NewSession(nil)
 	return conn, sess
 }
+
+func TestDoltServerRunningUnixSocket(t *testing.T) {
+	if runtime.GOOS == "windows" {
+		t.Skip("we define socket flag and unix socket cannot be created on Windows, so it will return error")
+	}
+
+	// TODO : socket flag is defined in this startServer method for now
+	unixSocketPath := "/tmp/mysql.sock"
```

Comment:
(minor) To tidy up... you can:
* remove line 356 – I didn't think it was adding any extra info
* inline the `unixSocketPath` variable since it's only referenced once